src/WebJobs.Script/OutOfProc/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -211,6 +211,27 @@ public async Task InitializeAsync(IEnumerable<FunctionMetadata> functions)
             }
         }
 
+        public async Task ShutdownAsync()
+        {
+            _logger.LogDebug("Waiting for FunctionDispatcher to shutdown");
+            Task timeoutTask = Task.Delay(5000);
```

Comment:
make this constant. Also, is there a timeout on orphan old jobHost task? If yes, this should use the same timeout.

Reply:
yeah, It's 10 seconds. Updated this timeout to match the same